pkg/hubble/exporter/builder.go

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Hubble
+
+package exporter
+
+import (
+	"fmt"
+	"maps"
+	"slices"
+)
+
+// FlowLogExporterBuilder knows how to build a FlowLog The Replaces field
+// indicate that an existing builder with a matching Name field should be
+// replaced by this one.
+type FlowLogExporterBuilder struct {
+	Name     string
+	Replaces string
+	Build    func() (FlowLogExporter, error)
+}
+
+// resolveExporters verifies the slice of builders for duplicates and performs
+// replace operations as needed before building and returning the resulting
+// FlowLogExporters.
+//
+// This function panics if two or more builders have the same name or a
+// Replaces target builder was not found.
+func ResolveExporters(builders []*FlowLogExporterBuilder) ([]FlowLogExporter, error) {
+	builders, err := replaceBuilders(builders)
+	if err != nil {
+		// if we reach here, we got an implementation bug since we manage builders in code
+		panic(err)
```

Comment:
Could we maybe provide a better panic message?
```suggestion
		panic(fmt.Sprintf("cannot resolve flow exporter: %s": err))
```

Reply:
Done :)